spec/controllers/students_controller_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
+
+RSpec.describe StudentsController, type: :controller do
+    #save instance of student in test db 
+    let!(:student) { create(:student, title: "Mr", first_name: "Harry", middle_name: "James", last_name: "Potter", email: "harry@test.com", gender: "Male", birth_date: "1989-07-23") }
```

Comment:
@Luke-kb Always create your objects with a purpose in mind.
As a reviewer when I look at this student object, with Mr Harry James Potter .. etc etc
I would expect you to use all those values somewhere below in some context, however that are not used.

So be explicit: .. for this Student object .. who cares what the values are, let the faker gem take care of it.
```
let!(:student) { create(:student) }
```

Reply:
Ah I see. So use literal values as per the [recommendation](https://jobready.atlassian.net/wiki/spaces/DEV/pages/43941936/Code+Review+Checklist) only for the values that you intend to test?